src/identity-providers/IdentityProvidersSection.tsx

```diff
@@ -113,11 +110,18 @@ export const IdentityProvidersSection = () => {
           <DropdownItem
             key={provider.id}
             value={provider.id}
-            data-testid={provider.id}
-            onClick={() => navigateToCreate(provider.id)}
-          >
-            {provider.name}
-          </DropdownItem>
+            component={
+              <Link
+                to={toIdentityProviderCreate({
+                  realm,
+                  providerId: provider.id,
```

Comment:
Once `providerId` is required you can use the non-null assertion operator here.
```suggestion
                  providerId: providerId!,
```